From 363e41db3715c0a53ded5f3a58fae437281f6225 Mon Sep 17 00:00:00 2001
From: John Vogel <jvogel@stygian.me>
Date: Wed, 16 Sep 2026 13:36:38 -0400
Subject: [PATCH] contrib/lx/jgmenu-lx.c: adjust rtrim per src/util.[ch] change

Previously, the rtrim function took a char **. That has been
changed to char *, so adjust the call in contrib/lx/jgmenu-lx.c.
---
 contrib/lx/jgmenu-lx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/lx/jgmenu-lx.c b/contrib/lx/jgmenu-lx.c
index 5d00d90..497996c 100644
--- a/contrib/lx/jgmenu-lx.c
+++ b/contrib/lx/jgmenu-lx.c
@@ -112,7 +112,7 @@ static void strip_exec_field_codes(char **exec)
 				*p = ' ';
 		}
 	}
-	rtrim(exec);
+	rtrim(*exec);
 }
 
 /*
-- 
2.55.0

